### PR TITLE
SapMachine #1370: Fix Vitals: long term table does not show one sample per hour

### LINKED ISSUE #1370: Vitals: long term table does not show one sample per hour
In the long term table samples are added every 10 hours with VitalsSampleInterval default (10 seconds) only. Because the VitalsSampleInterval is not considered.
Caused by SampleTables changes with https://github.com/SAP/SapMachine/commit/a7782411e239b524db13c6d2ba8558d4a60f1cef

### PR DESCRIPTION
**Pull request by ansteiner**
Consider VitalsSampleInterval for adding sample to the long term table

fixes #1370

### CODE REVIEW

**Comment by SapMachine** [comment]
Hello @ansteiner, this pull request fulfills all formal requirements.

File: src/hotspot/share/vitals/vitals.cpp [hunk 1]
```diff
@@ -752,14 +752,14 @@ class PrintSamplesClosure : public SampleTable::Closure {
   }
 };
 
-// sampleTables is a combination of three tables: a short term table, a mid term table, a long term table.
```
Comment (RealCLanger) [inline src/hotspot/share/vitals/vitals.cpp:755]: nit: a short term table and a long term table

**Review by RealCLanger — CHANGES_REQUESTED** [review]
Technically looks correct. Two suggestions regarding comments...

**Comment by SapMachine** [comment]
Hello @ansteiner, this pull request fulfills all formal requirements.

File: src/hotspot/share/vitals/vitals.cpp (hunk 1 above)
Comment (ansteiner) [reply src/hotspot/share/vitals/vitals.cpp:755]: done

**Review by RealCLanger — APPROVED** [review]

**Pull request merged**